api/v1beta2/gitrepository_types.go

```diff
@@ -227,6 +227,21 @@ type GitRepositoryStatus struct {
 	// +optional
 	ContentConfigChecksum string `json:"contentConfigChecksum,omitempty"`
```

Comment:
Would mark this with `// Deprecated`, this increases visibility of the deprecation in the more intelligent IDEs.

Reply:
Added. Thanks.